pkgs/development/libraries/at-spi2-atk/default.nix

```diff
@@ -1,24 +1,45 @@
-{ stdenv, fetchurl, python, pkgconfig, popt, atk, libX11, libICE, xorg, libXi
-, intltool, dbus-glib, at-spi2-core, libSM }:
+{ stdenv
+, fetchurl
+
+, meson
+, ninja
+, pkgconfig
+
+, at-spi2-core
+, atk
+, dbus
+, glib
+, libxml2
+
+, gnome3 # Only to pass update script
+}:
+
+with stdenv.lib;
 
 stdenv.mkDerivation rec {
-  versionMajor = "2.26";
-  versionMinor = "1";
-  moduleName   = "at-spi2-atk";
-  name = "${moduleName}-${versionMajor}.${versionMinor}";
+  name = "${pName}-${version}";
+  pName = "at-spi2-atk";
```

Comment:
Oh.
Better everyone use camelCase, for consistency. ;)

I wrote pName because it is more readable.

Eye mixes-up `name` and `pname` while scanning code. It is literal sub-string match.

`name` and `pName` are very distinct.

Reply:
Tough luck, it is quite widespread – Python packages even require it.

```
$ rg '\bpname =' -l | wc -l
1926
```